### PR TITLE
Try making @mikroskeem happier

### LINKED ISSUE #37: Docker Compose file?
Hey why is there no docker compose file.
This is how I usually always spool up my containers and there is none supplied.

Do you plan on adding one?

### PR DESCRIPTION
**Pull request by jtagcat**
Closes #37

### CODE REVIEW

**Comment by mikroskeem** [comment]
pls

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  journal:
+    internal: true
+  proxy:
+
+services:
+  journal:
+    build: https://github.com/inoda/journal.git#main
+    image: gh-inoda-journal
+    volumes:
+      - /media/jc/floppy/jounral/data:/app:rw
```
Comment (inoda) [inline docker-compose.yml:11]: Is there a typo here? Should this say `journal`?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - journal
+    restart: unless-stopped
+
+  journal-db:
+    image: postgres:alpine
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "rahvakunst"]
+    environment:
+      - POSTGRES_DB=kebab
+      - POSTGRES_USER=rahvakunst
```
Comment (inoda) [inline docker-compose.yml:32]: Same thing here re: replacing `rahvakunst` and `kebab`

**Review by inoda — CHANGES_REQUESTED** [review]
Left a few comments

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+services:
+  journal:
+    build: https://github.com/inoda/journal.git#main
+    image: gh-inoda-journal
+    volumes:
+      - /media/jc/floppy/journal/data:/app:rw
+    environment:
+      - RAILS_ENV=production
+      - SITE_DOMAIN=domain.tld
+      - DATABASE_URL=postgres://replaceme:RePLAcEpAsS@journal-db/kebab
```
Comment (inoda) [inline docker-compose.yml:15]: what is `kebab`?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      - proxy
+      - journal
+    restart: unless-stopped
+
+  journal-db:
+    image: postgres:alpine
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "replaceme"]
+    environment:
+      - POSTGRES_DB=kebab
```
Comment (inoda) [inline docker-compose.yml:31]: same here - what is `kebab`?

**Review by inoda — CHANGES_REQUESTED** [review]
still some strange placeholders

File: docker-compose.yml (hunk 3 above)
Comment (mikroskeem) [reply docker-compose.yml:15]: https://en.wikipedia.org/wiki/Kebab

I believe @jtagcat is trying to introduce better words than `foo`, `bar` and `baz`.

**Comment by wi0001** [comment]
🤔 🚀

**Pull request merged**